pkg/observ/observ.go

```diff
@@ -17,16 +18,56 @@ type observabilityContext struct {
 	spanCtx context.Context
 }
 
-// RegisterTraceExporter returns a jaeger exporter for exporting traces to opencensus.
+// RegisterExporter determines the type of TraceExporter service for exporting traces from opencensus
+// User can choose from multiple tracing services (datadog, jaegar)
+// RegisterExporter returns the 'Flush' function for that particular tracing service
+// TODO: Extend beyond jaeger and datadog
+func RegisterExporter(TraceExporter, URL, service, ENV string) (func(), error) {
+
+	const op errors.Op = "RegisterExporter"
+
+	switch TraceExporter {
+	case "jaeger":
+		ex, err := registerJaegerExporter(URL, service, ENV)
+		return jaegerFlush(ex), errors.E(op, err)
```

Comment:
Why are you always returning an error. 
https://github.com/gomods/athens/commit/0853f073f268e0b44eb75b2b45f3811b51054101

If I am reading the code correctly, error will never be nil.

I tried this :
`	fmt.Println("----------------", errors.E("manu", nil))`
and it returned 
`Internal Server Error`